Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -51,14 +56,20 @@ const run_tests = function (context) {
             });
         }
 
-        const config = {
+        let config = {
+            setupFilesAfterEnv:["<rootDir>/src/setupTests.js"],
```

Comment:
Unsure why, but if I remove this line I get the following:
![image](https://user-images.githubusercontent.com/66700089/132420439-23b3fc07-721b-40ee-b79d-7e243b5f03bf.png)